File: drawee/src/main/java/com/facebook/drawee/debug/listener/ImageLoadingTimeListener.java
/*
 * Copyright (c) Facebook, Inc. and its affiliates.
 *
 * This source code is licensed under the MIT license found in the
 * LICENSE file in the root directory of this source tree.
 */
package com.facebook.drawee.debug.listener;

/** Implement this interface to notify UI that the final Image has been set. */
public interface ImageLoadingTimeListener {
  void onFinalImageSet(long finalImageTimeMs);
}

File: drawee/src/main/java/com/facebook/drawee/debug/listener/ImageLoadingTimeListener.kt
/*
 * Copyright (c) Meta Platforms, Inc. and affiliates.
 *
 * This source code is licensed under the MIT license found in the
 * LICENSE file in the root directory of this source tree.
 */

package com.facebook.drawee.debug.listener

/** Implement this fun interface to notify UI that the final Image has been set. */
interface ImageLoadingTimeListener {
  fun onFinalImageSet(finalImageTimeMs: Long)
}
